xml/docu_styleguide.language.xml

```diff
@@ -76,6 +76,32 @@
   </sect3>
  </sect2>
 
+ <sect2 xml:id="sec-bias">
+  <title>Biases and Inclusivity</title>
+  <para>
+   Do not artificially limit your audience by excluding or offending members of
+   it.
+  </para>
+  <para>
+   Avoid indicating gender in your documentation.
+   If possible, use plural to allow use of <quote>they</quote> as the pronoun.
+   Otherwise, use <quote>he or she.</quote>
+  </para>
+  <para>
+   &suse; supports the Inclusive Naming Initiative which aims to help avoid
```

Comment:
```suggestion
   &suse; supports the <orgname>Inclusive Naming Initiative</orgname> which aims to help avoid
```

Reply:
That won't be formatted in the output in any way and I generally don't see the benefit unless we want some kind of machine process to run over this and do [$something].

Reply:
My idea was to make it visible that this is a proper name. Use literal or whatever, I don't care as long as it becomes visible (not just by the capitals)